### DISCUSSION TITLE
Add PKCE support in connected app authentication

### DISCUSSION THREAD

**Discussion by davescruggs** [forcedotcom/cli#2726] [Ideas]
PKCE is recommended now, but not supported for the CLI yet in a custom connected app: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_connected_app.htm. 

Can we add PKCE support for connected app authentication?

**Comment by shetzel** [comment]
The CLI always used to support this, but something got turned off.  This PR re-enables it: https://github.com/forcedotcom/sfdx-core/pull/1035

**Comment by davescruggs** [comment]
That is awesome

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jshackell-sfdc** [comment]
This issue is fixed in Salesforce CLI release 2.33.3.
